### DISCUSSION TITLE
How can I link to Projects from a note?

### DISCUSSION THREAD

**Discussion by marcusolsson** [marcusolsson/obsidian-projects#377] [FAQ]
Since Projects stores all its configuration in the plugin configuration, `data.json`, you can't link to a Project from a note using internal links.

A workaround is to use the [Advanced URI](https://github.com/Vinzent03/obsidian-advanced-uri) plugin, which lets you create links to commands.

1. Install [Obsidian Advanced URI](https://github.com/Vinzent03/obsidian-advanced-uri) from Community plugins.
2. In the **Command palette**, select **Advanced URI: copy URI for command**.
3. Select **<Don't specify a file>**.
4. Select **Projects: Show projects**.
5. In a note, enter `[Show projects](<paste the copied URI>)`.

From 1.13.0, you can also enable commands to show specific projects and views. To enable a command:

1. Open **Settings**.
2. Select **Projects** in the sidebar.
3. Under **Commands**, enable the projects and views you want to create commands for.

The command palette now has an additional Show command for each project and view that you enabled. Note that you can use **Advanced URI: copy URI for command** to create links for each one.

**Comment by Xelbayria** [comment]
Just getting my understanding correct, 

> you can't link to a Project from a note using internal links.

Does that mean one can't add an internal link (pointing at a Project) inside the note? 

---
It's the opposite of your topic. 
I am also here for answers: Is it possible to link a heading from just one note via Projects (Using Table view)?

**Comment by mbelyayev** [comment]
A feature request would be to add a command **Projects: Show Project `Project Name`**. Which essentially allows one to open a specific project, rather than the default project. 

So for example if you had created projects "A" and "B", the available commands would be:
**Projects: Show Project A**
**Projects: Show Project B**


This way people can link to different projects in rather than only the default project.

**Reply by Acylation** [reply]
Direct to the plugin settings page and find the `Commands` section, you can add these commands by simply switch them on. (There are also commands for views) FYI #216 and its solution #459

![image](https://github.com/user-attachments/assets/7f8a1919-e869-4380-83d6-6b0705167b14)

**Reply by whdmike** [reply]
I have a situation. I mark my projects as shown above (under commends) but somehow they switch off (I have them around 20) when I close settings, and they are not shown inside command pallete.

**Comment by stoweboyd** [comment]
You can follow @marcusolsson 's instructions for Advanced URI, and then store the URIs in metadata, too. I create a new field in many projects, add a new field to reference 'up' to the project, and use the URI as a default value. The, I can click 'up' to the project from any project note without havinf to run the command manually.

**Reply by pragmaticanD** [reply]
Hi @stoweboyd, can you elaborate what you store in the new field to go 'up'
I find it cumbersome to get the Advanced URI for each project view I want to link to. Would be nice if I can programmaticaly reference a project or particular view of a project inside a dataview query so my projects dashboard could be jump off point.
